scipy/fft/_debug_backends.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+try:
+    import pyfftw.interfaces.numpy_fft as pyfftw_fft
+except ImportError:
+    pyfftw_fft = {}
```

Comment:
From a licensing standpoint I'm a bit uneasy about importing anything from `pyfftw` in the SciPy module itself, even if it's only imported on demand. It might be better to stick this in with the `benchmarks` where you might actually use it (?).

But maybe I'm being too paranoid. @rgommers do you know the limitations here?

Reply:
pyFFTW/pyFFTW#229 may be relevant here.

Since it's not greatly important either way, I've moved it into the benchmarks as suggested.